Jakefile.js

```diff
@@ -1305,7 +1305,7 @@ task("lint", ["build-rules"], () => {
     
     const files = fileMatcher
         ? `src/**/${fileMatcher}`
-        : `Gulpfile.ts scripts/generateLocalizedDiagnosticMessages.ts "scripts/tslint/**/*.ts" "src/**/*.ts" --exclude "src/lib/*.d.ts"`;
+        : "Gulpfile.ts 'scripts/*generateLocalizedDiagnosticMessages.ts' 'scripts/tslint/**/*.ts' 'src/**/*.ts' --exclude 'src/lib/*.d.ts'";

```

Comment:
the string literal did not cause any issues right?

Reply:
The type of quotes seemed to matter.  In any case, I thought it would be easiest to just revert to a known state and then apply a new workaround.